orc8r/cloud/go/services/ctraced/obsidian/handlers/handlers.go

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handlers
+
+import (
+	"net/http"
+
+	"magma/orc8r/cloud/go/obsidian"
+	"magma/orc8r/cloud/go/orc8r"
+	"magma/orc8r/cloud/go/serdes"
+	"magma/orc8r/cloud/go/services/configurator"
+	"magma/orc8r/cloud/go/services/ctraced/obsidian/models"
+	"magma/orc8r/cloud/go/services/ctraced/storage"
+	merrors "magma/orc8r/lib/go/errors"
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/labstack/echo"
+	"github.com/pkg/errors"
+)
+
+const (
+	Tracing = "tracing"
```

Comment:
minor nit: make the constants not used outside this module as private.

Reply:
Okay